src/lib.rs

```diff
@@ -2943,11 +2943,39 @@ impl Connection {
         {
             if let Some(max_dgram_payload) = self.dgram_max_writable_len() {
                 while let Some(len) = self.dgram_send_queue.peek_front_len() {
-                    if (len + frame::MAX_DGRAM_OVERHEAD) <= left {
```

Comment:
you remove use of frame::MAX_DGRAM_OVERHEAD here but it is still used at L2940, is that intentional?

Reply:
Yeah, we need _some_ value there, and we don't know what the front DATAGRAM's size is. We can probably rework this code a bit more to remove the constant from there too, but it seems better to leave that for a future change to avoid making this too big.

Reply:
WFM